hs_labels/tests/__init__.py

```diff
@@ -1,2 +1,2 @@
 from __future__ import absolute_import
-from .test_resource_labelling import *
\ No newline at end of file
+#from .test_resource_labelling import *
```

Comment:
I commented this for testing purpose. The tests in hs_labels are running twice on Jenkins due to this import. But I also see some other tests running twice without this import statement. So not what's going on. However, in my final commit for this PR this code statement will not remain as commented.